api/src/main/java/org/openmrs/util/ConceptReferenceRangeUtility.java

```diff
@@ -0,0 +1,165 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.util;
+
+import org.apache.velocity.VelocityContext;
+import org.apache.velocity.app.VelocityEngine;
+import org.openmrs.Obs;
+import org.openmrs.Person;
+import org.openmrs.api.ObsService;
+import org.openmrs.api.ValidationException;
+import org.openmrs.api.context.Context;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.StringWriter;
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.temporal.ChronoUnit;
+
+/**
+ * A utility class that evaluates the concept ranges 
+ */
+public class ConceptReferenceRangeUtility {
+	
+	private static final Logger logger = LoggerFactory.getLogger(ConceptReferenceRangeUtility.class);
+	
+	public ConceptReferenceRangeUtility() {
+	}
+	
+	/**
+	 * This method evaluates the given criteria against the provided context.
+	 *
+	 * @param criteria the criteria string to evaluate e.g. "$fn.getAge($patient, 'YEARS') > 1 && $fn.getAge($patient, 'YEARS') < 10"
+	 * @param person person object containing variables to be used in the criteria
+	 * @return true if the criteria evaluates to true, false otherwise
+	 */
+	public static boolean evaluateCriteria(String criteria, Person person) {
```

Comment:
Is there any reason why you made the methods in this class static?

Reply:
Since this is a utility class, the methods shouldn't require an instance of the class. I have however removed static from other methods.

Reply:
@dicksonmulli do you get slack notifications?

Reply:
> Since this is a utility class, the methods shouldn't require an instance of the class. I have however removed static from other methods.

But your intended use in the velocity context requires an instance. So design them according to your intended usage.